packages/core/src/package-utils/dts.ts

```diff
@@ -59,5 +62,12 @@ export const createDtsBundle = async (
 
   await bundle.close();
 
+  await promiseMap(entries, async (entry) => {
+    await copyFile(
+      path.join(entry.outputDir, entry.getOutputPath('dts')),
+      path.join(entry.outputDir, entry.getOutputPath('dtsm')),
+    );
+  });
+
```

Comment:
Both of these `copyFile`s just appeared, so I'm a bit confused to where the copy was happening before this change, i.e. the one that copied just the `.d.ts` file?

Reply:
It's just one `copyFile(/* from */ dts, /* to */ dtsm)`.

Reply:
Oh it is too 😮‍💨. It looked like 2 because my small window size was causing it to wrap onto a bunch of extra lines.